elements/rh-accordion/rh-accordion-header.ts

```diff
@@ -35,7 +35,9 @@ export class RhAccordionHeader extends BaseAccordionHeader {
 
   static readonly styles = [...BaseAccordionHeader.styles, styles];
 
-  @colorContextConsumer() private on: ColorTheme = 'light';
+  @colorContextConsumer()
+  @property({ reflect: true })
```

Comment:
Why does this need to reflect?

Reply:
This brought up a good question I had yesterday that I wasn't able to figure out till this morning.  Previously I had issues getting the attributes to pass down to the child components so I added the reflects back and it worked.  It looks like though I would only need that on the `rh-accordion` element parent but the `panel` just needed a few css updates & a wrapper to have the class internally and it didn't need to reflect anymore either.